src/Camelot.Services.Abstractions/ILanguageManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using Camelot.Services.Abstractions.Models;
+
+namespace Camelot.Services.Abstractions
+{
+    public interface ILanguageManager
+    {
+        LanguageModel GetCurrentLanguage { get; }
```

Comment:
I think it's better to avoid actions as names for properties, it's good name for method but not for property